packages/components/tests/dummy/app/templates/components/alert.hbs

```diff
@@ -0,0 +1,147 @@
+{{page-title "Alert component"}}
+
+<h2 class="dummy-h2">Alert</h2>
+
+<section>
+  <h3 class="dummy-h3">Scratch reference links</h3>
+  <p>Alert types coming soon:
+    <code class="dummy-code">Hds::Alert::FullWidth</code>,
+    <code class="dummy-code">Hds::Alert::Inline</code>, and
+    <code class="dummy-code">Hds::Alert::Toast</code>.
+  </p>
+  <ul class="dummy-paragraph">
+    <li>
+      <a href="https://docs.google.com/document/d/1MC0MTm8-0F26s6Y-gou_RsMj6rtKS6jC-eM7rmruqLY/edit#heading=h.gjdgxs">
+        WIP Alert Component Requirement Document (CRD)
+      </a>
+    </li>
+    <li>
+      <a
+        href="https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/branch/Ru0h1oqy6r7SeGxwtQ6085/HDS-Product---Components-%5BWIP%5D?node-id=4023%3A15691"
+      >
+        WIP Figma file
+      </a>
+    </li>
+  </ul>
+</section>
+
+<section>
+  <h3 class="dummy-h3">Component API</h3>
+  <p class="dummy-paragraph">Here is the API for the component:</p>
+  <dl class="dummy-component-props">
+    <dt>color <code>enum</code></dt>
+    <dd>
+      <p><code class="dummy-code">color</code>
+        results in a default
+        <code class="dummy-code">icon</code>, which
+        <strong>can</strong>
+        be overriden.
+        <code class="dummy-code">color</code>
+        sets the color scheme for background, border,
+        <code class="dummy-code">title</code>, and
+        <code class="dummy-code">description</code>, which
+        <strong>cannot</strong>
+        be overriden.
+      </p>
+      <p>Acceptable values:</p>
+      <ol>
+        <li>critical</li>
+        <li>warning</li>
+        <li class="default">neutral</li>
+        <li>highlight</li>
+        <li>success</li>
+      </ol>
+    </dd>
+
+    <dt class="dummy-indent">icon <code>string</code></dt>
+    <dd class="dummy-indent">
+      <p>Shows an icon in the alert at a set size of 24 viewBox. Optional parameter.</p>
+      <p>Acceptable value:</p>
+      <p>Any
+        <a href="https://flight-hashicorp.vercel.app/" target="_blank" rel="noopener noreferrer">
+          Flight icon</a>
+        name.
+      </p>
+    </dd>
+
+    <dt>title <code>string</code></dt>
+    <dd>
+      <p>
+        The title text in the alert.
+        <code class="dummy-code">title</code>
+        is required.
+      </p>
+    </dd>
+
+    <dt>description <code>string</code></dt>
+    <dd>
+      <p>The description text in the alert.
+        <code class="dummy-code">description</code>
+        is optional.</p>
+    </dd>
+
+    <dt>isDismissible <code>boolean</code></dt>
+    <p>The alert can be dismissed by the user. This parameter has some accessibility considerations for
```

Comment:
@amyrlam apologies for commenting on a draft but I think this will save you some time in the long run- the only eligible items inside of a `<dl>` element are `<dt>` and `<dd>` - any `<p>` elements should go inside of the `<dd>` element. 👍

Reply:
@MelSumner thanks - I'll fix that. Also please feel free to comment on any PR, is helpful!

Reply:
@amyrlam please don't fix it for now. we've used this pattern across every documentation page. if we'll do the change,  we'll do it in another PR across the entire documentation (possibly in the new version of the documentation website, for now, for the scrappy website, it's OK to leave it as is).

Reply:
There was some confusion, fixed it here https://github.com/hashicorp/design-system/pull/99/commits/4d3492ea00896c43b44fa6c6e2b9b6a29b8dd756, left a note here https://app.asana.com/0/1201687774781544/1201999776475344/f